lib/stacker/cli.rb

```diff
@@ -111,18 +115,6 @@ def dump stack_name = nil
       end
     end
 
-    desc "fmt [STACK_NAME]", "Re-format template JSON"
-    def fmt stack_name = nil
```

Comment:
AWS CFN supports YAML and stores as YAML if that's what you give it. We can leave it, but it'll only apply to JSON templates, which makes t a quirky command.